tests/functional/monitoring/conftest.py

```
@@ -1181,25 +1184,57 @@ def measure_change_client_ocs_version_and_stop_heartbeat(
-        client.set_ocs_version("4.13.0")
-        logger.info(f"Waiting for {run_time} seconds")
-        time.sleep(run_time)
+        # Wait for heartbeat critical alert to fire (300s + margin)
+        heartbeat_wait_time = 60 * 6  # 6 minutes
+        logger.info(f"Waiting {heartbeat_wait_time} seconds for heartbeat to be missed")
+        time.sleep(heartbeat_wait_time)
+
+        # Calculate incompatible version (1 minor version lower)
+        from ocs_ci.utility.version import get_previous_version
+        incompatible_version = get_previous_version(current_version, count=1)
+        logger.info(f"Setting incompatible version: {incompatible_version} (current: {current_version})")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)